core/Pos/Core/Constants/Typed.hs

```diff
@@ -61,6 +61,7 @@ genesisBlockVersionData =
     , bvdUpdateProposalThd = genesisUpdateProposalThd
     , bvdUpdateImplicit = genesisUpdateImplicit
     , bvdUpdateSoftforkThd = genesisUpdateSoftforkThd
+    , bvdTxFeePolicy = Nothing
```

Comment:
I think it should be configurable by `constants.yaml`. Probably it's for CSL-1242, so ok.